gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManagerMetrics.java

```diff
@@ -212,6 +216,13 @@ public void incrementFailedLaunchCount() {
     }
   }
 
+  // Increment the count for num of successful launches attempted during leader activation
+  public void incrementSuccessfulLaunchAttemptCount() {
```

Comment:
I see the comment above (should be javadoc), but the method name invites confusion, since it sounds like this is for counting successes, irrespective of whether or not during startup

(BTW, "system startup" is slightly more accurate for our multi-leader arch than "leader activation" )